lib/utils/runOpen.js

```diff
@@ -12,7 +12,11 @@ function runOpen(uri, options, log) {
     openMessage += `: ${options.open}`;
   }
 
-  return open(`${uri}${options.openPage || ''}`, openOptions).catch(() => {
+  const pageUrl = /^(https?:)?\/\//.test(options.openPage)
+    ? options.openPage
+    : `${uri}${options.openPage || ''}`;
+
+  return open(pageUrl, openOptions).catch(() => {
     log.warn(
       `${openMessage}. If you are running in a headless environment, please do not use the --open flag`
```

Comment:
I think that error message should be updated to also reflect hints of failures when using a non valid absolute url

Reply:
no failures when using non valid absolute url it'll append it to the server.
ex: openPage is ht:/nonvalid and the devserver is http://localhost:8080 
the result will be  http://localhost:8080/ht:/nonvalid

Reply:
👍